backend/compact-connect/lambdas/cloudfront-csp/index.js

```diff
@@ -17,29 +17,34 @@ const environments = {
         prod: {
             webFrontend: `app.compactconnect.org`,
             dataApi: `api.compactconnect.org`,
-            s3Upload: ``, // @TODO: Waiting for environment configuration.
+            s3Upload: `prod-persistentstack-bulkuploadsbucketda4bdcd0-zq5o0q8uqq5i.s3.amazonaws.com`,
+            cognitoStaff: `compact-connect-staff.auth.us-east-1.amazoncognito.com`,
         },
         test: {
             webFrontend: `app.test.compactconnect.org`,
             dataApi: `api.test.compactconnect.org`,
-            s3Upload: `test-persistentstack-mockbulkuploadsbucket0e8f27eb-ivslg22f2yfz.s3.amazonaws.com`,
+            s3Upload: `test-persistentstack-bulkuploadsbucketda4bdcd0-gxzuwbuqfepm.s3.amazonaws.com`,
+            cognitoStaff: `compact-connect-staff-test.auth.us-east-1.amazoncognito.com`,
         },
     },
     ia: {
         prod: {
             webFrontend: `app.jcc.iaapi.io`,
             dataApi: `api.jcc.iaapi.io`,
-            s3Upload: ``, // @TODO: Waiting for environment configuration.
+            s3Upload: ``, // Waiting for environment configuration.
+            cognitoStaff: ``, // Waiting for environment configuration.
         },
         test: {
             webFrontend: `app.test.jcc.iaapi.io`,
             dataApi: `api.test.jcc.iaapi.io`,
-            s3Upload: `test-persistentstack-mockbulkuploadsbucket0e8f27eb-4h1anohxetmp.s3.amazonaws.com`,
+            s3Upload: `test-persistentstack-bulkuploadsbucketda4bdcd0-er1izmgsrdva.s3.amazonaws.com`,
+            cognitoStaff: `ia-cc-staff-test.auth.us-east-1.amazoncognito.com`,
         },
         justin: {
             webFrontend: `app.justin.jcc.iaapi.io`,
             dataApi: `api.test.jcc.iaapi.io`,
             s3Upload: `test-persistentstack-mockbulkuploadsbucket0e8f27eb-4h1anohxetmp.s3.amazonaws.com`,
+            cognitoStaff: ``, // Waiting for environment configuration.
         },
```

Comment:
Also, small nit, but ideally, this repository is agnostic to the fact that IA is its author. Is there any (reasonable) way to keep these variables outside of the codebase (particularly for the `ia` and `justin` environments)?

Reply:
Yeah, we sometimes go back and forth with this. Ideally (and to your point), every last bit of config would live outside the code. The problems we've run into in the past with that approach are that more in-flux items can halt progress over days (or sometimes weeks) while configs are coordinated between teams / platforms. We usually try to limit the cases of config-in-code to where the risk of time lost to coordination is higher and the code smell is more acceptable. Open to additional thoughts.

Reply:
Got it. I lean towards moving config outside of code, but not without a more thorough discussion about what that would entail for you all (pros, cons, and all). Could the application not just read a json config file that lives somewhere else? Why can't this data live in `cdk.context.json`, for example?

Let's have this discussion separately, though, and proceed with this approach for now.

Reply:
Sounds good. For standard Lambdas, the solution would probably be environment variables; however these are Edge@Lambdas (distributed to the CDN edges) and AWS doesn't allow dynamic environment variables on those. So I think solutions like a JSON file (or secrets service) would require us to create a build step, where the config is either bundled with the function code (slower run-time perf) or injected as static code (faster run-time perf) before CDN distribution. Those are additional moving parts that we don't currently have implemented. I'll create a ticket to track the discussion.

Reply:
Sounds good. May or may not be relevant but from a quick google search, this may be another option (I'm obviously less well-versed in the details of our implementation, though, so not sure if this solution applies to us): https://stackoverflow.com/questions/54828808/aws-lambdaedge-nodejs-environment-variables-are-not-supported